ui/imports/shared/views/chat/ChatContextMenuView.qml

```diff
@@ -276,6 +276,29 @@ StatusMenu {
         }
     }
 
+    Component {
+        id: leaveGroupConfirmationDialogComponent
+        ConfirmationDialog {
+            confirmButtonObjectName: "leaveGroupConfirmationDialogLeaveButton"
+            headerSettings.title: qsTr("Leave group")
+            confirmationText: qsTr("Are you sure you want to leave group chat with <b>%1</b>?").arg(root.chatName)
```

Comment:
The group chat name can be changed to something else than the member's name, so maybe saying `you want to leave group chat <b>%1</b>?"` might be better